db/patch-081_draft.sql

```diff
@@ -0,0 +1,76 @@
+-- Associated with BWD 205 Create a new Health Impacts Function group called “Chronic Effects - Primary”. 
+-- Add the following HIFs from the current “Chronic Effects - All”:
+-- Ozone: Tetrault asthma, Parker hay fever/rhinitis
+-- PM: Wei AMI, Tetrault asthma, Parker hay fever/rhinitis, Gharibvand lung cancer,Ensor OOH cardiac arrest,Rosenthal OOH cardiac arrest, Silverman OOH cardiac Arrest, Kloog stroke, Wilker et al., new dementia function
+-- UPDATE "data".settings SET value_int=64 WHERE "key"='version'; --need to update version when this is approved
+
+
+UPDATE "data".settings SET value_int=81 where "key"='version';
+
+
+INSERT INTO data.health_impact_function_group("name", "help_text", "share_scope") 
+      select   'Chronic Effects - Primary', 'Primary funtions selected from the larger chronic effects group', 1
+where not exists (
+  select 1 from data.health_impact_function_group where name = 'Chronic Effects - Primary'
+);
+
+
+-- insert into data.health_impact_function_group_member("health_impact_function_group_id","health_impact_function_id")
```

Comment:
This is the original query to get the list of IDS listed below that fit the criteria to go under the new hif_group "Chronic Effects - All"